site/docs/hosting/firebase.md

```diff
@@ -0,0 +1,186 @@
+# Hosting: Firebase Functions
+
+This tutorial will guide you through the process of deploying your bot to [Firebase Functions](https://firebase.google.com/docs/functions).
+
+## Prerequisites
+
+To follow along, you will need to have a Google account.
+If you don't already have one, you can create it [here](https://accounts.google.com/signup).
+
+## Setup
+
+This section guides you through the setup process.
+If you need more detailed explanations on each step that you'll take, refer to the [official Firebase documentation](https://firebase.google.com/docs/functions/get-started).
+
+### Create a Firebase Project
+
+1. Go to the [Firebase console](https://console.firebase.google.com/) and click **Add Project**.
+2. If prompted, review and accept the Firebase terms.
+3. Click **Continue**.
+4. Decide on whether you want to share analytics or not.
+5. Click **Create Project**.
+
+### Setup Node.js and Firebase CLI
+
+To write functions and deploy them to the Cloud Functions runtime, you'll need to set up a Node.js environment and install the Firebase CLI.
+
+> It's important to note that only Node.js versions 14, 16, and 18 are currently supported by Firebase Functions.
+> For more information on Node.js version support, please refer to [Set Node.js version](https://firebase.google.com/docs/functions/manage-functions#set_nodejs_version) section in their docs.
+
+Once you have Node.js and npm installed, you can install the Firebase CLI using your preferred method.
+To install it via npm, use the following command in your terminal:
+
+```sh
+npm install -g firebase-tools
+```
+
+### Initialize The Project
+
+1. Run `firebase login` to log in via the browser and authenticate the Firebase CLI.
```

Comment:
```suggestion
1. Run `firebase login` to open your browser and authenticate Firebase CLI.
```

Reply:
The Browser opens automatically and asks for your credentials / which account to use, if you are already logged in your google account